src/elarm_config.erl

```diff
@@ -81,25 +81,26 @@ get_unconfigured(#config_state{ missing_tab = MissingTab } = State) ->
 
 %% Get a list of all alarm_ids that have mapping configuration.
 -spec get_configured(#config_state{}) ->
```

Comment:
The spec in line 83 uses a record, please define a type for the record and use that instead.